CONTRIBUTING.md

```diff
@@ -167,3 +167,71 @@ git push && git push --tags
 
 This will make a commit with the updated `package.json`, create a new git tag, and publish to npm.
 Make sure you are logged in to npm, talk to a maintainer.
+
+## Code overview
+
+### HTTP Client
+
+The core of the SDK is the HTTP client. The HTTP client is divided into multiple layers:
+
+1. [BasicHttpClient](./packages/core/src/httpClient/basicHttpClient.ts)
+2. [RetryableHttpClient](./packages/core/src/httpClient/retryableHttpClient.ts)
+3. [CDFHttpClient](./packages/core/src/httpClient/cdfHttpClient.ts)
+
+See each file for a description of what they do.
+
+### Pagination
+
+We have multiple utilities to easy pagination handling. The first entrypoint is [cursorBasedEndpoint](./packages/core/src/baseResourceApi.ts) which adds a `next()` function on the response to fetch the next page of result. Then we use [makeAutoPaginationMethods](./packages/core/src/autoPagination.ts) to add the following methods:
+
+- autoPagingToArray
+  ```ts
+  const assets = await client.assets.list().autoPagingToArray({ limit: 100 });
+  ```
+- autoPagingEach
```

Comment:
Something is up with the markdown rendering from this point onwards. Could you see if that can be fixed by changing something with the triple-tick blocks?

Reply:
<img width="715" alt="Screenshot 2024-09-23 at 15 28 30" src="https://github.com/user-attachments/assets/ab4d1518-8651-4893-85ce-c9c17e58e5f8">
LGTM?